v03_pipeline/lib/misc/family_loading_failures.py

```diff
@@ -99,7 +111,9 @@ def build_relatedness_check_lookup(
         j=remap_lookup.get(relatedness_check_ht.j, relatedness_check_ht.j),
     )
     return {
-        (r.i, r.j): list(r.drop('i', 'j').values())
+        # NB: samples are sorted in the original ibd but not necessarily
```

Comment:
what's ibd?

Reply:
It's the relatedness matrix produced by [`hl.identity_by_descent`](https://hail.is/docs/0.2/methods/relatedness.html#hail.methods.identity_by_descent).  There's some examples of what it looks like there or in our tests.